Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN apk --no-cache add \
 COPY . /fiaas-deploy-daemon
 COPY .wheel_cache/*.whl /links/
 WORKDIR /fiaas-deploy-daemon
+RUN pip install -U setuptools
```

Comment:
maybe there is a better way?

Reply:
Maybe it is a good idea to pin the setuptools version ? (and possibly also include a comment for the future about why this is necessary?) It looks like setuptools [45.0.0 drops python2 support](https://setuptools.readthedocs.io/en/latest/history.html#v45-0-0), so [44.1.1](https://pypi.org/project/setuptools/44.1.1/) should be the most recent setuptools version that supports python2.

Reply:
it makes totally sense, I pinned the version and added some comments